src/models/ankle_draft.js

```diff
@@ -0,0 +1,90 @@
+import { pmv } from "./pmv.js";
+import {
+  units_converter,
+  check_standard_compliance,
+  round,
+} from "../utilities/utilities.js";
+
+/**
+ * @public
+ * @typedef {Object} Ankle_draft_ret
+ * @property {number} PPD_ad - redicted Percentage of Dissatisfied occupants with ankle draft, [%]
+ * @property {boolean} Acceptability - The ASHRAE 55 2020 standard defines that the value of air speed at the ankle level is acceptable if PPD_ad is lower or equal than 20%
+ */
+
+/**
+ * Calculates the percentage of thermally dissatisfied people with the ankle draft (0.1 m) above floor level {@link #ref_23|[23]}.
+    This equation is only applicable for vr < 0.2 m/s (40 fps).
+ * @public
+ * @memberof models
+ * @docname Ankle draft
+ * 
+ * @param {number} tdb - dry bulb air temperature, default in [°C] in [°F] if `units` = 'IP'   
+ * 
+        Note: The air temperature is the average value over two heights: 0.6 m (24 in.)
+        and 1.1 m (43 in.) for seated occupants
+        and 1.1 m (43 in.) and 1.7 m (67 in.) for standing occupants.
+
+ * @param {number} tr - mean radiant temperature, default in [°C] in [°F] if `units` = 'IP'
+
+ * @param {number} vr - relative air speed, default in [m/s] in [fps] if `units` = 'IP'   
+
+        Note: vr is the relative air speed caused by body movement and not the air
+        speed measured by the air speed sensor. The relative air speed is the sum of the
+        average air speed measured by the sensor plus the activity-generated air speed
+        (Vag). Where Vag is the activity-generated air speed caused by motion of
+        individual body parts. vr can be calculated using the function v_relative in utilities.js.
+
+ * @param {number} rh - relative humidity, [%]
+
```

Comment:
why so many empty lines in the docs?

Reply:
I intend to widen the horizontal gap between documentation lines. But looks it didn't work